component/azstorage/connection.go

```diff
@@ -136,6 +136,10 @@ type AzConnection interface {
 	TruncateFile(string, int64) error
 	StageAndCommit(name string, bol *common.BlockOffsetList) error
 
+	GetCommittedBlockList(string) (*internal.CommittedBlockList, error)
+	StageBlock(string, []byte, string) error
+	CommitBlocks(string, []string) error
```

Comment:
We should add tests for these 3 methods in azstorage component. Not in this PR, maybe after the track2 migration.

Reply:
With dummy storage we can not do these tests, Will check if its possible to do this as part of this PR itself.

Reply:
I would suggest doing this in some other PR or maybe after the track2 migration is done, I will add these tests.